### PR TITLE
update user-profile/index.md

### PR DESCRIPTION
**Pull request by hillaryfraley**
Suggest changes to enhance clarity, accuracy, and completeness.

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/user-profile/index.md [hunk 1]
```diff
@@ -4,114 +4,116 @@ url: /user-profile
...
 
-Within the Dashboard, you may create, view, modify, or delete a given user. These options are available to you via the [Users tab](${uiURL}/#/users).
+Within the Dashboard, you can create, view, modify, or delete users. These options are available via the [Users](${uiURL}/#/users) page.
 
 ![](/media/articles/user-profile/user-profile-dashboard.png)
 
 ### Creating a New User
 
-To create a new user and their corresponding user profile, click on the "+ Create User" button at the top right of the Users page in the Management Dashboard. You will be asked to provide the following pieces of information:
+To create a new user and their corresponding User Profile, click the [Users](${uiURL}/#/users) page in the Dashboard. Click the "+ Create User" button at the top right of the Users page. You will be prompted to enter the following information:
```
Comment (Catografix) [inline articles/user-profile/index.md:23]: Button text looks better in **bold** instead of "quotes".

File: articles/user-profile/index.md [hunk 2]
```diff
@@ -4,114 +4,116 @@ url: /user-profile
...
 
-A popup displays the URL to be used in order to impersonate the user. You can choose either to copy the URL into the clipboard (white button) or open it in a separate browser tab/window (blue button).
+A popup will display the URL to use to impersonate the user. You can copy the URL into the clipboard (white button) or open the URL in a separate browser tab/window (blue button).
 
 ![](/media/articles/user-profile/signin-as-user-02.png)
 
-> You can also use the [Impersonation API](/api/authentication#!#post--users--user_id--impersonate). The API generates a link that can be used once to log in as a specific user. To distinguish between real logins and impersonation logins, the profile of the impersonated user will contain additional `impersonated` and `impersonator` properties.
+> You can also use the [Impersonation API](/api/authentication#!#post--users--user_id--impersonate) to impersonate a user. The API generates a link that you can used once to log in as a specific user. To distinguish between real logins and impersonation logins, the profile of the impersonated user will contain additional `impersonated` and `impersonator` properties.
```
Comment (Catografix) [inline articles/user-profile/index.md:95]: This and the following paragraph might be better in a panel.
::: panel-info Impersonation API
You can also use the [Impersonation API](/api/authentication#!#post--users--user_id--impersonate) to impersonate a user. The API generates a link that you can used once to log in as a specific user. To distinguish between real logins and impersonation logins, the profile of the impersonated user will contain additional `impersonated` and `impersonator` properties. Please see [User Impersonation](/user-profile/user-impersonation) for more information about impersonating a user.
:::

**Comment by mpaktiti** [comment]
Thank you for the contribution @hillaryfraley ! 👍

**Comment by hillaryfraley** [comment]
Oh, my pleasure! It was fun. Thank you for the opportunity!

**Pull request merged**